basics/installation/install-with-docker.md

```diff
@@ -0,0 +1,471 @@
+---
+title: Installing PrestaShop with Docker
+menuTitle: Installation with Docker
+weight: 15
+---
+
+# Installing PrestaShop with Docker
+
+PrestaShop is available through Docker for testing/experimenting, and developping modules or themes. 
+
+Docker is an open-source platform that enables developers to package applications and their dependencies into lightweight, portable containers, providing an efficient and consistent way to build, ship, and run software across different environments.
+
+- [Docker's website](https://www.docker.com/)
+- [PrestaShop's Docker Github repository](https://github.com/PrestaShop/docker)
+- [PrestaShop's Docker Hub](https://hub.docker.com/r/prestashop/prestashop/)
+
+The easiest way to begin with PrestaShop on Docker is to run your stack with Docker compose. It will ease container management, networks, volume persistence, ...
```

Comment:
```suggestion
The easiest way to begin with PrestaShop on Docker is to run your stack with Docker Compose. It will ease container management, networks, and volume persistence.
```